backend/app/src/main.ts

```diff
@@ -0,0 +1,10 @@
+import { NestFactory } from "@nestjs/core";
+import { AppModule } from "./app.module";
+
+async function bootstrap()
+{
+	const app = await NestFactory.create(AppModule);
+	await app.listen(3000);
+}
+
+bootstrap();

```

Comment:
knowing that bootstrap is a library, I think renaming this function would make more sense.

Reply:
I see bootstrap staff make a good communication.
This is the nestjs's function architecture.
Here boostrap means (Dictionnary's definition) : a technique of loading a program into a computer by means of a few initial instructions which enable the introduction of the rest of the program from an input device.